### PR TITLE
feat: Style valid and invalid examples

### PR DESCRIPTION
**Pull request by littletof**
Before / After:
image

### CODE REVIEW

**Comment by bartlomieju** [comment]
Looks good! Thanks for looking into this @littletof. @hashrock @ije can you review?

**Comment by bartlomieju** [comment]
Also @littletof how does it look in dark mode?

**Comment by littletof** [comment]
@bartlomieju No problem.

Right, I haven't looked into dark mode, and it looks bad. But I think It has other problems with darkmode (at least when emulating preference). 

Will look into it.

image

**Comment by bartlomieju** [comment]
Thank you, much appreciated

**Comment by littletof** [comment]
@bartlomieju Dark mode could look like this:

image

I will open a separate PR for the whole darkmode fix, to get merged before this.

File: www/pages/index.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ function IndexPage(props: PageProps) {
     useData("www/public/docs.json", diskFetcher),
   );
 
+  const stlye = useData("www/public/extension.css", diskFetcher);
```
Comment (hashrock) [inline www/pages/index.tsx:19]: stlye -> style

File: www/pages/index.tsx [hunk 2]
```diff
@@ -44,6 +46,7 @@ function IndexPage(props: PageProps) {
           href="https://cdn.jsdelivr.net/gh/lucacasonato/manual@df7ae27/www/static/markdown.css"
           crossOrigin="anonymous"
         />
+        <style>{stlye}</style>
```
Comment (hashrock) [inline www/pages/index.tsx:49]: stlye -> style

**Review by hashrock — COMMENTED** [review]
almost looks great!

**Comment by hashrock** [comment]
image

Not related to this PR, but we'd like to use classes instead of ids, since there can be more than one element on the same page. let's fix it in another PR.

File: www/pages/index.tsx (hunk 1 above)
Comment (littletof) [reply www/pages/index.tsx]: Right, rookie mistake. Fixed.

**Comment by littletof** [comment]
@hashrock Yes, first I wanted use classes in the selector, and it would be better, but these sections are generated from markdown, and to properly change them, we would need to extend the markdown -> html generator and/or add some custom markers in the markdown descriptions. (eg. replace mardown headers with HTML header tags in the markdown itself)

As a hotfix we can string replace all `id=invalid` to `class=invalild`  before we insert the HTML string, but thats not the nicest 🙂

**Comment by hashrock** [comment]
I see...thanks. I think I'll leave it as is for now.

**Pull request merged**